runtime/src/accounts_db.rs

```diff
@@ -260,7 +262,7 @@ pub struct AccountsDB {
     pub storage: RwLock<AccountStorage>,
 
     /// distribute the accounts across storage lists
-    next_id: AtomicUsize,
+    pub next_id: AtomicUsize,
```

Comment:
why is this pub, now?

Reply:
The next_id needs to be initialized when the bank snapshots are restored and is set to the value of the next snapshot id which is used as the next AppendVecId to use. This value is set from the bank currently, if not pub would need to expose a set routine to set this value from accounts and accounts_db.